CHANGELOG.md

```diff
@@ -16,6 +16,8 @@
   * Various functions in the `math_utils` crate can now return a `MathError` : `div_mod`, `ec_add`, `line_slope`, `ec_double`, `ec_double_slope`.
   * Fixes `UINT256_MUL_INV_MOD_P` hint so that it behaves like the python code.
 
+* feat: Added a differential fuzzer for programs with whitelisted hints
+
```

Comment:
This should go at the top as it will be the most recent change once merged, also, include a link to this PR

Reply:
fixed! https://github.com/lambdaclass/cairo-vm/pull/1358/commits/09c95d157047ddbea17fb33130c671ae30c2e639